src/main/java/frc/controllers/TeleopControls.java

```diff
@@ -16,12 +16,16 @@ public TeleopControls(Drives drives){
         
         Joystick joystick = new Joystick(0);
         spinButton = new Button(joystick, 0);//Creates button for keeping track of button 0
+        backwardButton = new Button(joystick, 2);
```

Comment:
I had to add a private variable to make mine work, I suggest you add one to yours.